.github/workflows/on-pull-request-run-tests.yml

```diff
@@ -3,6 +3,9 @@ name: On Pull Request - Run Tests
 on:
   pull_request:
 
+env:
+  IS_GITHUB_ACTION: true
```

Comment:
why would we only check for these variables in github actions?

Reply:
Idea was to skip that check in GithubActions.